### DISCUSSION TITLE
What kind of Sentry project should I use for Cloudflare Workers?

### DISCUSSION THREAD

**Discussion by chaodoze** [getsentry/support#8] [Support] [answered]
Unfortunately, I don't see Cloudflare Workers listed under "Serverless" in projects, and I couldn't get @sentry/browser to work. Thanks!

**Comment by souredoutlook** [comment, marked as answer]
Hi @chaodoze thanks for your patience!

We don't have 1st-party support for Cloudflare workers at this time. 

I found this recent blog post [about using toucan.js with Cloudflare workers](https://surendharreddy.dev/2022/03/06/cf-workers-logging/). I can see that the repo is well maintained.

I'll update this Discussion if I find a better solution.
